yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -43,11 +43,6 @@ class FlowableOperator : public Flowable<D> {
         : flowable_(std::move(flowable)), subscriber_(std::move(subscriber)) {
       assert(flowable_);
       assert(subscriber_);
-
-      // We expect to be heap-allocated; until this subscription finishes (is
-      // canceled; completes; error's out), hold a reference so we are not
-      // deallocated (by the subscriber).
-      Refcounted::incRef(*this);
```

Comment:
I believe this class was historically written as a copy/paste of SynchronousSubscription which has this refcounting logic and its necessary. In this case, since the OperatorSubscription is always held by the reference, I agree that it is unnecessary.